firefox_puppeteer/api/places.py

```diff
@@ -4,6 +4,8 @@
 
 from collections import namedtuple
 
+from marionette_driver.errors import MarionetteException
```

Comment:
You missed to address this comment. Can you please do it, squash again, and force push? Once done I will merge your PR. Btw. the line I was referencing it line 8 now.